classes/classes/Scenes/NPCs/JoyScene.as

```diff
@@ -514,7 +514,7 @@ package classes.Scenes.NPCs
 			outputText("You decide to ask Joy what she thinks about you.");
 			outputText("\n\nJoy blinks and looks at you puzzled. \"<i>What do I, like, think of you? Okay...</i>\" She narrows her eyes, studying you intensely.\n\n");
 			//Race check
-			switch(player.race) {
+			switch(player.race()) {
```

Comment:
While this is pedantic, the change is unrelated to the rest of the commit.
Probably would have been better to include in a separate commit.